test/release/examples/primers/learnChapelInYMinutes.chpl

```diff
@@ -682,7 +682,7 @@ writeln(1 * 2);   // Uses the default ``*`` operator.
 
 /* .. code-block:: chapel
 
-      proc +(left: int, right: int): int {
+      operator +(left: int, right: int): int {
```

Comment:
I feel like this should be `operator int.+` because the `bool.^` example above is tied to `bool`?

Reply:
I suggest that we do not turn numeric and bool operators into methods, generally speaking. For one, there are lots of operators in ChapelBase that would have to be switched to methods.

Moreover, there are some operators, say, on int that we can't turn into methods on `int` because they are already methods on `codepointIndex` or `byteIndex` etc. If those work fine without being methods on `int`, why do any other int operators need to be methods?

The same argument perhaps applies to `string`, `byte`, `c_string`, maybe some others.

Reply:
It's arguably a style thing (especially in this test) but I would agree that it's better to use non-method operators for `bool` and `int`.